Packages/Editor/Sources/Editor/EditorViewController.swift

```diff
@@ -281,6 +301,25 @@ public final class EditorViewController: UIViewController {
     private func shiftDown() {
         textView.moveSelectedLinesDown()
     }
+
+    private func performImageUpload(base64Image: String) {
+        Task { @MainActor in
+            let textController = TextController(textView: textView)
+            textController.imageUploadPlacholder()
+            do {
+                let imageData = try await viewModel.uploadImage(base64Image: base64Image)
+                textController.insertImage(url: imageData.link)
+            } catch {
+                showErrorAlert(title: error.localizedDescription)
+            }
+        }
+    }
+
+    private func showErrorAlert(title: String) {
+        let alert = UIAlertController(title: title, message: nil, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "OK", style: .cancel))
+        present(alert, animated: true)
+    }
```

Comment:
We are using `AlertToast` to show error. Can you make a wrapper version for UIKit, you can follow `DesignSystem` package